### ISSUE TITLE
Feature request: Throttle and/or pause Comfyui mid-generation.

### ISSUE THREAD

**Issue by sgsrules** [comfyanonymous/ComfyUI#2645]
I'm running some animatediff worklows that can take a whole day to process. If i need my gpu for anything else i either have to wait for it to complete or cancel it and restart the process. This makes me lose any progress on whatever node it was currently on which could cost me hours.

It would be very useful to be able throttle performance so that the gpu is not pegged the whole time. 

Being able  pause mid generation and then unpause later would also be hugely beneficial.

**Comment by Hangover3832** [comment]
Indeed, a pause function would be useful sometimes. As a workaround in the meantime, selecting some text in the ComfyUI console with the mouse does the job. Then press <ESC> to continue generation.

**Comment by Satellile** [comment]
Agreed that this would be useful.

As an alternative workaround for Windows Terminal, select the terminal ComfyUI is running in, and press PauseBreak on your keyboard to halt its execution. Then press Ctrl+Z to unpause and resume generation.

I also made [a simple command-line tool](https://github.com/Satellile/yara) that can save your queued up prompts to disk. You can then load them later, allowing you to close ComfyUI completely and resume generating when you're ready. But it can't save partial progress on a single prompt, so it's mostly of interest if you want to pause a large number of queued up prompts (while it sounds like your situation is just one very long prompt).

**Comment by mysticfall** [comment]
I have RTX3080, which frequently shuts down while generating due to overheating. This feature would be beneficial for me to throttle or pause it when it becomes too hot.

**Comment by c0dezer019** [comment]
I fourth this.
